package.json

```diff
@@ -122,7 +127,8 @@
     "ts-node": "^10.9.1",
     "typed-css-modules": "^0.7.0",
     "typed-css-modules-loader": "0.0.18",
-    "typescript": "^4.9.5"
+    "typescript": "^4.9.5",
+    "webpack": "^4.46.0"
```

Comment:
Do we need this installed?

I uninstalled it and it still worked

Reply:
hmmm... you's right!